tools/ops/README.md

```diff
@@ -11,6 +11,18 @@ The only required argument is `-c`: a list of config (policy) files.
 $ python tools/ops/mugc.py -c policies.yml
 ```
 
+The policy file must be like:
+
+```
+policies:
+  - name: delete
+    resource: ebs
+```
+
+**You can't choose which CloudCustodian lambda will be delete**
```

Comment:
you can actually, --prefix or --regex can point to the full name of the lambda

Reply:
Oh great, I will add more info for `--prefix`
I don't see any references to `--regex` in the setup function:
```
def setup_parser():
    parser = argparse.ArgumentParser()
    parser.add_argument("configs", nargs='*', help="Policy configuration file(s)")
    parser.add_argument(
        '-c', '--config', dest="config_files", nargs="*", action='append',
        help="Policy configuration files(s)", default=[])
    parser.add_argument(
        '-r', '--region', action='append', dest='regions', metavar='REGION',
        help="AWS Region to target. Can be used multiple times, also supports `all`")
    parser.add_argument('--dryrun', action="store_true", default=False)
    parser.add_argument(
        "--profile", default=os.environ.get('AWS_PROFILE'),
        help="AWS Account Config File Profile to utilize")
    parser.add_argument(
        "--prefix", default="custodian-",
        help="The Lambda name prefix to use for clean-up")
    parser.add_argument("-p", "--policies", default=None, dest='policy_filter',
                        help="Only use named/matched policies")
    parser.add_argument(
        "--assume", default=None, dest="assume_role",
        help="Role to assume")
    parser.add_argument(
        "-v", dest="verbose", action="store_true", default=False,
        help='toggle verbose logging')
    return parser
```

Reply:
its a recent addition on trunk see policy-regex https://github.com/cloud-custodian/cloud-custodian/blob/master/tools/ops/mugc.py#L171

Reply:
oohhh, I'm out to date, sorry.